external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MsSqlServerIntegrationSuite.scala

```diff
@@ -202,4 +202,46 @@ class MsSqlServerIntegrationSuite extends DockerJDBCIntegrationSuite {
     df2.write.jdbc(jdbcUrl, "datescopy", new Properties)
     df3.write.jdbc(jdbcUrl, "stringscopy", new Properties)
   }
+
+  test("Write tables with ShortType") {
```

Comment:
I think this is a bug, so can you append the JIRA ID in the prefix?

Reply:
@maropu is JIRA addition required at all places i have added a new test cases?

Reply:
Yea, plz follow `test("SPARK-29644: test title") `